index.html

```diff
@@ -23,16 +23,26 @@
 <body>
```

Comment:
Please add your report to the description of your pull request so that a reviewer can check the points you write on your project accessablity.